tests/functional/i/implicit/implicit_str_concat_latin1.py

```diff
@@ -1,4 +1,5 @@
 # coding: latin_1
+# -1: [bad-file-encoding]
 #pylint: disable=invalid-name,missing-docstring
 
-TOTO = ('Caf�', 'Caf�', 'Caf�')
+TOTO = ('Caf�', 'Caf�', 'Caf�')
```

Comment:
Is it normal that the encoding was changed here ?

Reply:
No good catch. I have to revert the file partly. Probably an issue because PyCharm didn't detect the correct encoding and then I pressed save, trusting that not changed chars would remain the same

Reply:
fixed